sql/sqlite/migrate.go

```diff
@@ -344,7 +344,32 @@ func (s *state) copyRows(from *schema.Table, to *schema.Table, changes []schema.
 			fromC = append(fromC, column.Name)
 		}
 	}
-	stmt := fmt.Sprintf("INSERT INTO %s (%s) SELECT %s FROM %s", to.Name, strings.Join(toC, ", "), strings.Join(fromC, ", "), from.Name)
+
+	var toCols string
+	for i, cc := range toC {
+		if strings.Contains(cc, "`") {
+			toCols += cc
+		} else {
+			toCols += "`" + cc + "`"
+		}
+		if i+1 < len(toC) {
+			toCols += ", "
+		}
+	}
+
+	var fromCols string
```

Comment:
The code for `toC` was correct in the previous commit, but for `fromC` is actually more tricky because it can contain expression and a placeholder (see, L332). 

Maybe add the backticks when appending the columns? L330, L339, L343-4.

Reply:
Re-reading it now, the code is correct. Another approach is to move it to a private function and reuse it here:

```go
func identComma(c []string) string {
	b := &sqlx.Builder{QuoteChar: '`'}
	b.MapComma(c, func(i int, b *sqlx.Builder) {
		if strings.ContainsRune(c[i], '`') {
			b.Ident(c[i])
		} else {
			b.WriteString(c[i])
		}
	})
	return b.String()
}
```

Reply:
yea I didn't have time to deep dive into the code so I went with the least invasive option. let me know if anything else needs to be adjusted.